example/.eslintrc

```diff
@@ -13,6 +13,9 @@
         "expect": true,
         "protractor": true
     },
+    "rules": {
+        "prefer-arrow-callback": 0
+    },
```

Comment:
Just pushed all the changes to all the comments a second ago 👍  They don't appear in this view though, seems like you have to click on the "Files" tab in Github to see them